### PR TITLE
es6-operators - Tim Silva

### PR DESCRIPTION
**Pull request by silva-tim**
Screen Shot 2023-06-20 at 2 46 59 PM

### CODE REVIEW

File: es6-operators/main.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+const v4 = [3, 5, 7, 9];
+
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
+ * In one expression, log "v1 is truthy" to the console only if `v1` is truthy.
+ * In one expression, log "v2 is truthy" to the console only if `v2` is truthy.
+ * Expected: v1 message logs, v2 message does not.
+ */
+console.log(v1 && 'v1 is truthy');
+console.log(v2 && 'v2 is truthy');
```
Comment (shawnkost) [inline es6-operators/main.js:14]: We should only be console logging if `v1` or `v2` is truthy. Right now your 2nd console log is still firing, it's just not printing any value

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Tim!

File: es6-operators/main.js (hunk 1 above)
Comment (silva-tim) [reply es6-operators/main.js]: Fixed

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**